src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -76,7 +77,8 @@
~        return variant.value == "test" && !settings.scanningReminderWasShown
     }
 
```

Comment:
wow! didn't know that trick, how did you found it? can't find it in official docs....